ethereum/handler/ethereumHandler.go

```diff
@@ -163,7 +166,10 @@ func (eh *EthereumHandler) PrintInfo(ctx context.Context, seconds time.Duration)
 		info, err = eh.SyncInfo(tctx)
 		cancel()
 		if err != nil {
-			log.Warn(err)
+			log.Warnf("error getting ethereum info: %s", err)
+			if err := eh.Connect(eh.Endpoints); err != nil {
+				log.Warnf("cannot connect to any web3 endpoint: %v", err)
```

Comment:
Shouldn't this Fatal?

Reply:
yes assuming last comment made on this, changed to a fatal